algorithms/48/fib.cpp

```diff
@@ -1,17 +1,16 @@
 #include <iostream>
 using namespace std;
 
-// returns the nth fibonacci number
-long long fib(int n) {
-    if(n == 0) return 0;
-    if(n == 1) return 1;
-    return fib(n-1)+fib(n-2);
-}
 
 int main() {
     int n;
     cout << "Enter a number: ";
     cin >> n;
-    cout << "The " << n << "th fibonacci number is " << fib(n) << endl;
+	long long fib[n+1];
+	fib[1]=0;
+	fib[2]=1;
+	for(int i=3;i<=n;i++)
+		fib[i]=fib[i-1]+fib[i-2];
```

Comment:
In the original code, 
```
fib(0) -> 0
fib(1) -> 1
fib(2) -> 1
fib(3) -> 2
```

However, in your code it is,
```
fib(0) -> 0
fib(1) -> 0
fib(2) -> 1
fib(3) -> 1
```

Could you change your code to match the same output as the original one?